src/vnstatd.c

```diff
@@ -233,7 +233,7 @@ void showhelp(void)
 	printf("      -D, --debug              show additional debug and disable daemon\n");
 	printf("      -?, --help               show this help\n");
 	printf("      -v, --version            show version\n");
-	printf("      -p, --pidfile <file>     select used pid file\n");
+	printf("      -p, --pidfile <file>     select used pid file when running in background mode\n");
```

Comment:
I feel this addition is not necessary and it's better to keep the help texts short

Reply:
Agreed, especially as the next commit points handles this.